view/frontend/web/js/view/checkout/summary/normalize-cart.js

```diff
@@ -39,7 +39,9 @@ define([
       template: 'Extend_Integration/checkout/summary/normalize-cart',
     },
     shouldNormalize: function () {
-      return true
+      if (window.checkoutConfig.extendEnable === '1') {
```

Comment:
I do still see the templates loading for `shipping-protection.html`, `shipping-protection-offer.html` and `normalize-cart.html` but assuming we are not intending to keep the elements from loading just prevent them from taking any actions?

Reply:
as long as they're not communicating externally, or causing errors

Reply:
Unless we need it even more disabled, @jm-extend ?

Reply:
we can test. i assume we have higher level checks (window.checkoutConfig.extendEnable === '1') in place too right ?  the thought being that we might not even run these templates ?